docs/upgrading.md

```diff
@@ -12,6 +12,10 @@ Migration guides contain information regarding the new features introduced in ea
 
 ## General Information
 
+Upgrading is not advised on running apps. Stop them before with `dokku ps:stop <app>`.
```

Comment:
It should be noted that docker >=1.11 promises your containers don't need to be restarted if the docker daemon is restarted. However live upgrades in a production level environment (without fail overs) are not recommended.